UI/css/ledgersmb.css

```diff
@@ -5,6 +5,22 @@ td.money {
    text-align: right;
 }
 
+td.date {
+   white-space: nowrap!important;
+}
+
+td.check {
+    text-align: center;
+}
+
+div#title {
+    font-size: large;
+}
+
+th.header {
```

Comment:
Actually, isn't TH *always* used as a header in our code? That's its semantic meaning, isn't it? Shouldn't we then default to centered, with resetting to "initial" on more specific use cases?